controllers/delete.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { getCommentById } = require("../db/seeds/utils")
+const {removeCommentById} = require("../models/delete.models")
+
+exports.deleteCommentById = (req, res, next) => {
+    const comment_id = req.params.comment_id
+
+    getCommentById(comment_id).then((existingComment) => {
```

Comment:
You don't **need** to get the comment first to prove it exists before trying to delete it, if you just go straight to trying to remove a comment by an id and from the RETURNING * the rows are empty - you can infer that it doesn't exist. What you've done isn't wrong though could just be handled differently if you like

Reply:
I don't know if I'm supposed to reply here but I was struggling to get a 404 error if I didn't prove it existed first. So I did this to fix that. How else would I go about getting the 404?